src/lib/y2network/widgets/wireless_auth_mode.rb

```diff
@@ -0,0 +1,46 @@
+require "cwm/common_widgets"
+
+module Y2Network
+  module Widgets
+    class WirelessAuthMode < CWM::ComboBox
+      def initialize(settings)
+        textdomain "network"
+
+        @settings = settings
+      end
+
+      def init
+        self.value = @settings.auth_mode
+      end
+
+      def label
+        _("&Authentication Mode")
+      end
+
+      def opt
+        [:hstretch, :notify]
+      end
+
+      def items
+        [
```

Comment:
Constant?

Reply:
well, when I start working on master mode then it need to be dynamic, as master mode does not support all auth modes.